portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -400,6 +417,14 @@ definition {
 			expected = "${managerId2}");
 	}
 
+	macro assertResponseNotIncludeDetailsOfDeletedObject {
```

Comment:
I have asked to look for `universitiesSubjects` so it returns empty lists. I'm afraid that if we compare the specific values to an empty String it might be a bit misleading (not sure what will happen if the structure of the response changes in the future or if any of the names of the field we're checking). So, knowing that we should have received empty objects in all these cases, we're already checking this level. Does that make sense?